### PR TITLE
Remove use of hard-coded system property strings

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

File: grakn-core/src/main/java/ai/grakn/GraknSystemProperty.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import javax.annotation.Nullable;
+
+/**
+ * @author Felix Chapman
+ */
+public enum GraknSystemProperty {
+
+    CURRENT_DIRECTORY("grakn.dir"),
+    CONFIGURATION_FILE("grakn.conf"),
+    GRAPH_CONFIG_PROPERTY("grakn.conf"),
```
Comment (alexandraorth) [inline grakn-core/src/main/java/ai/grakn/GraknSystemProperty.java]: Why are there two properties that are the same?

Comment (aelred) [inline grakn-core/src/main/java/ai/grakn/GraknSystemProperty.java]: all better

**Comment by aelred** [comment]
!rtg - spooky cassandra problem. filipe on high alert

File: grakn-core/src/main/java/ai/grakn/GraknSystemProperty.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ *
+ */
+
+package ai.grakn;
+
+import javax.annotation.Nullable;
+
+/**
+ * @author Felix Chapman
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/GraknSystemProperty.java]: Bad javadocs are bad. We should elaborate the purpose of this class.

**Comment by aelred** [comment]
@fppt I added javadocs, but Github doesn't seem to care? :/

Undeniable proof: https://github.com/aelred/grakn/commits/system-properties

**Pull request merged**